src/MyBrokenClass.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class MyBrokenClass
+{
+    protected string $test;
+
+    public function test(): string
+    {
+        foreach ( $_POST['menu-locations'] as $menu_id ) {
+            echo 'ADD warning severity 10 - non-sanitized input variable';
+        }
+        $functionsCookies = $_COOKIE;
+
+        $functionsPost = $_POST;
+        var_dump($functionsPost);
+
+        session_write_close();
```

Comment:
:no_entry_sign: **Error**: The use of PHP session function session_write_close() is prohibited (*WordPressVIPMinimum.Functions.RestrictedFunctions.session_session_write_close*).